src/board/system76/darp8/board.mk

```diff
@@ -7,12 +7,13 @@ EC=ite
 CONFIG_EC_ITE_IT5570E=y
 
 # Enable eSPI
-CFLAGS+=-DEC_ESPI=1
-# Apply PMC hack for S0ix
-CFLAGS+=-DPMC_S0IX_HACK=1
+CONFIG_BUS_ESPI=y
+CFLAGS+=-DCONFIG_BUS_ESPI=1
```

Comment:
This CFLAG addition could be added in common.mk when CONFIG_BUS_ESPI=y is set

Reply:
Done